resources/lang/de/partials.php

```diff
@@ -35,7 +35,7 @@
   'area_text_1' => 'Through this community, potential volunteers self-register and share their location. The platform is designed to connect organisers and fixers.',
   'area_text_2' => 'Through this community, potential volunteers self-register and share their location. Here\'s a list of potential volunteers near you',
   'upcoming_events' => 'Upcoming events',
-  'no_upcoming_events' => 'No Upcoming Events',
+  'no_upcoming_events' => 'No upcoming Events',
```

Comment:
If it's changing to lower case, events should also be lower case, no?

Reply:
That looks like a search and replace mistake - there's no reason to make this change to the German version if we're not making it to the English version.  Thanks for spotting.